### PR TITLE
bugfix & setup to replace 3p helpers incrementally

### PR DESCRIPTION
**Pull request by bc-evan-johnson**
## What? Why?

The previous iteration of these helper replacements broke backwards compatibility in a couple of previously-unknown edge cases.

Note: The helper replacements will be enabled in future PRs.

## How was it tested?

Additional unit test cases have been added. Both the new and old versions pass the new test cases.

----

cc @bigcommerce/storefront-team

### CODE REVIEW

File: spec/helpers/option.js [hunk 1]
```diff
@@ -21,22 +21,23 @@ describe('option helper', function () {
...
 
-    it('accepts SafeString paths', (done) => {
-        runTestCases([
-            {
-                input: `{{option (concat 'a.b.' 'c')}}`,
-                output: `d`,
-            },
-        ], done);
-    });
+    // it('accepts SafeString paths', (done) => {
```
Comment (jairo-bc) [inline spec/helpers/option.js:35]: Shouldn't it be supported now?

File: helpers.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const helpersList = [
     'dynamicComponent',
     'encodeHtmlEntities',
     'for',
-    'get',
+    // 'get',
```
Comment (jairo-bc) [inline helpers.js:14]: Lets maybe enable 1 helper here? Maybe `get`

File: spec/helpers/option.js (hunk 1 above)
Comment (bc-evan-johnson) [reply spec/helpers/option.js:35]: The old `option` helper (and also `get`) only works with SafeString for top-level properties, not nested properties, so this test will fail on the old version.

File: helpers.js (hunk 2 above)
Comment (bc-evan-johnson) [reply helpers.js:14]: Let's start with `moment` and `option`.

**Review by jairo-bc — APPROVED** [review]
